lib/index.js

```diff
@@ -34,11 +34,14 @@ module.exports = {
     }
 
     var createPackageTree = function (depTree) {
+      var targetFramework = depTree.meta ? depTree.meta.targetFramework : undefined; // TODO implement for paket and more than one framework
```

Comment:
Maybe `let` or `const`?

Reply:
can't cause of node 4

Reply:
updated in later review to `strict` mode for now